modules/Planner/planner_add.php

```diff
@@ -1220,6 +1220,8 @@ function outcomeDisplayElements(number) {
 							<span class="emphasis small">* <?php echo __($guid, 'denotes a required field'); ?></span>
 						</td>
 						<td class="right">
+							<input type="checkbox" name="notify" value="on">
+							<label for="notify">Notify all class participants</label>
```

Comment:
Wrapping the text inside this label with `__()` will tell the code that it's translatable. Generally, anytime you write a string that the user will read it's best to add __().